source/_patterns/20-molecules/menu-list/menu-list.scss

```diff
@@ -0,0 +1,9 @@
+.menu-list {
+  margin: 0;
+  padding: 0;
+}
+
+.menu-list__item {
```

Comment:
nitpicking - könntest du auch als
```

.menu-list,
.menu-list__item {
  ...
}
```

schreiben. Aber ist auch OK so

Reply:
ist angepasst.

eigentlich bin ich ja mit dem ganzen element `menu-list`nicht happy - aber irgendwie muss ich die paddings und margins in der liste loswerden